content/docs/dvclive/dvclive-with-dvc.md

```diff
@@ -7,22 +7,26 @@ ways:
   `dvc metrics` and `dvc plots`. Those same outputs can be visualized in
   [Iterative Studio](#iterative-studio).
 
-- You can monitor model performance in realtime with the
-  [HTML report](#html-report) that DVCLive generates when used alongside DVC.
-
 - DVCLive is also capable of generating [checkpoint](#checkpoints) signal files
```

Comment:
"several useful ways" now down to 2. Should we say "a couple useful ways" or expand the list?